specs/protocol/stage-1.md

```diff
@@ -42,3 +44,105 @@ not be able to perform any actions which have an impact on either the safety or
 system. An example of such a role is the `SystemConfig` owner (AKA Chain Operator), which can
 modify fees and other protocol parameters. Constraints MUST be placed on the allowable range of
 values for these parameters so that they do not result in a loss of liveness or safety.
+
+## Configuration of Safes
+
+This list outlines the various Safes, their extensions, and other configuration details necessary to
+achieve Stage 1.
+
+1. **The ProxyAdminOwner Safe:** This Safe acts as the Upgrade Controller. Accordingly, it is
+   authorized to call the following safety-critical functions:
+      - All `ProxyAdmin` `onlyOwner` functions.
+      - All `DisputeGameFactory` `onlyOwner` functions.
+      - All `DelayedWeth` `onlyOwners` functions.
+
+   This safe has a threshold of 2, and is owned by two other Safes:
+      1. The Security Council Safe.
+      2. The Foundation Upgrade Safe.
+
+   In general, the threshold and owner set of the safe must be configured such that an upgrade
+   or other safety-critical action can be performed without the cooperation of the Security Council.
+
+1. **The Guardian Safe:** This Safe
+   includes but is not limited to pausing all code paths related to withdrawals. It is also extended
+   with the [Deputy Guardian Module](./safe-extensions.md#deputy-guardian-module).
+
+   Accordingly, this Safe is authorized to call the following liveness-critical functions:
+      - All `SuperchainConfig` `onlyOwner` functions.
+      - All `OptimismPortal2` `onlyOwner` functions.
+
+   This Safe has a threshold of 1 and is owned by the Security Council Safe.
+
+1. **The Security Council Safe:** This Safe is one of the two owners of the `ProxyAdminOwner` Safe.
+   It is extended with the [Liveness Checking system](./safe-extensions.md#liveness-checking-system),
+   although Liveness Checking is not required for Stage 1, it is an additional safety feature used
+   to prevent any loss of liveness on the Security Council Safe.
+
+   The threshold and owner set of the Safe MUST meet L2Beat's Stage 1 requirements.
+
+1. **The Foundation Upgrade Safe:** This Safe is one of the two owners of the `ProxyAdminOwner` Safe.
+   It is also able to update the recommended and required versions on the `ProtocolVersions`
+   contract, given that observing the state of this contract is optional, this is not considered to
+   be affect safety and can therefore be managed the Foundation Safe.
+
+1. **The Foundation Operations Safe:** This Safe acts as the Deputy Guardian, meaning that (via the
+   Guardian Safes's `DeputyGuardianModule`) it can call any functions in the system which impact
+   liveness.
```

Comment:
Similar "liveness" question as the ones in the comments of https://github.com/ethereum-optimism/specs/pull/333

Reply:
resolved in #333